doc/src/selections.rst

```diff
@@ -27,15 +27,22 @@ of three bonded atoms forming an angle such that the name of the second atom is
 These constraints are created using *selectors*. Selectors are small functions
 which can be applied to one or more atoms. The ``name``, ``mass`` and ``z``
 selectors are applied to one atom and give the name, the mass and the z
-coordinate of this atom. Other selectors can be applied to two or more atoms,
-like ``distance`` or ``angle``. When using a selection with more than one atom,
+coordinate of this atom. When using a selection with more than one atom,
 selectors refers to the different atoms with the ``#1``, ``#2``, ``#3`` or
 ``#4`` variables: ``name(#3)`` will give the name of the third atom, and so on.
 Then we compare the selector result for a given atom with a value, either from
 another selector --- ``name(#1) != name(#2)`` --- or a literal value ---
 ``mass(#2) < 4``. The comparison operators are ``==`` (equals); ``!=`` (not
-equals); ``<`` (less than), ``<=`` (less or equals); ``>`` (more than); and ``>=``
-(more or equals).
+equals); ``<`` (less than), ``<=`` (less or equals); ``>`` (more than); and
+``>=`` (more or equals).
+
+Numeric values produced either by numeric selectors (``x``; ``mass``, ...) or
+literal values (``5.2``, ``22.21e-2``) can also be combined together using
+mathematical operations: the usual ``+``, ``-``, ``*`` and ``/`` operators are
+supported, as well as ``^`` for power raising. Some mathematical functions are
```

Comment:
I prefer "exponents" over "power raising," but that's a personal preference.

Reply:
Maybe also mention the order of operations here. Is 2+3*3 evaluate to 11 or 18?

Reply:
2+3*3 evaluate to the expected 11 (and not 15, nor 18 ^^) here, but yes I can mention it 😃